entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+testConnection="/opt/mssql-tools/bin/sqlcmd -S sql -U sa -P Password123"
+
+until $testConnection; do
+>&2 echo "SQL Server is starting up.."
+sleep 1
+done
```

Comment:
Is there a chance this could result in an infinite loop if there is some other remote server problem?